app/controllers/users_controller.rb

```diff
@@ -11,7 +12,8 @@ def new
   def create
     @user = User.new user_params
     if @user.save
-      flash[:success] = t ".welcome"
+      UserMailer.account_activation(@user).deliver_now
+      flash[:warning] = "Please check your email to activate your account."
```

Comment:
Viết gộp vào dòng dưới `redirect_to root_path, warning: "Please check your email to activate your account"`